src/main/scala/feria/Feria.scala

```diff
@@ -47,6 +50,11 @@ object Feria extends App {
       if (driver.getCurrentUrl.contains("AccountChooser"))
           driver.findElementByXPath("//button[contains(@value,'guardian.co.uk')]").click()
 
+      config.alias.filter(_.nonEmpty).foreach { alias =>
+        val profileField = driver.findElement(By.id("aws-profile-id"))
+        profileField.clear()
```

Comment:
I think this will explode (NullPointerEx?) in the case where you're not logged in to Janus and you get the Google login page. Could you move it inside the Try below?